lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -414,6 +415,9 @@ sub set_common_sdaf_os_env {
     $args{sdaf_tfstate_storage_account} //= get_required_var('SDAF_TFSTATE_STORAGE_ACCOUNT');
     $args{sdaf_key_vault} //= get_required_var('SDAF_KEY_VAULT');
 
+    # This is used later filling up tfvars files.
+    set_var('SDAR_REGION_CODE', $args{sdaf_region_code});
```

Comment:
there was no any `tfvars` before that needs to know about region? Is there any other setting about region?

Reply:
It's in the public IP and NAT GW resource name :

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19461/files#diff-f72c92100b8dddfc10ceb79d42a1fb58d2bf3de5d03c946fbcf85e51506526fbR96

But maybe I could try to remove the whole naming line. With other resources the naming is done internally.

Reply:
So on last MS meeting they mentioned that naming is mandatory. :/